CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [v0.12.0] - Unreleased
+## [v0.12.0] - 2023-11-XX
```

Comment:
Is the XX really intended?

Reply:
No its a todo... we don't know the date yet... I will wait with the release until we get confirmation that master image works as expected.